<!DOCTYPE html>
<html lang="en">


  <head>
    
    <link rel="stylesheet" href="main.css?v=<?=time();?">
    
  </head>


  <body>





    <div class="grid-container">


      <div class="picture">	
			 <div class="nav">
          <IMG SRC="pic.png" height="330">
			 </div>			      
      </div>

      
	           <div class="header">
<h1> Salil Bhate </h1>


<p> I am an <a href = "https://www.broadinstitute.org/ewsc">Eric and Wendy Schmidt Center</a> Postdoctoral Fellow at the Broad Institute,  working with Juan Caicedo and Caroline Uhler. 


<p> I work on formal tools to represent phenotypic information (e.g.,  <a href = "bhate et al. 2021 tissue schematics.pdf">tissue organization</a>) and its <a href = "https://www.biorxiv.org/content/10.1101/2022.08.15.503769v1">encoding</a>  in the genome.</p> 
			   
<p>I obtained my PhD (with <a href = "https://purl.stanford.edu/yc661nd1828">thesis</a> titled "Towards semantic representations of tissue organization from high-parameter imaging") in Bioengineering at Stanford University, where my advisor was Garry Nolan. A summary of this research can be found in my  <a href="https://www.broadinstitute.org/talks/towards-semantic-representations-tissue-organization-high-parameter-imaging-data">talk</a>  at the Broad's Models, Inference and Algorithms Initiative. </p>

<p>Before Stanford, I did my undergraduate/master's in pure math(s) at the University of Cambridge and some synthetic biology.</p>

<p>Here is my google scholar <a href="https://scholar.google.com/citations?user=F8PKG8QAAAAJ">profile</a> 
  and <a href="cv_aug22.pdf">CV</a> [last updated August '22].</p>
			   
			   <p> I currently co-chair the Steering Committee of <a href ="https://www.broadinstitute.org/talks/spring-2023/mia"> Models, Inference and Algorithms</a>. Please join! </p>



</div>
    </div>




    <div class="grid-container">

      <div class="picture">
      <div class="tab">
			     
  <button class="tablinks", onclick="opentab(event, 'Research Articles')">Research Articles</button>
  <button class="tablinks", onclick="opentab(event, 'Other Work')">Other Work</button>
  <button class="tablinks", onclick="opentab(event, 'Activities')">Activities</button>
  <button class="tablinks", onclick="opentab(event, 'Teaching')">Teaching</button>
    <button class="tablinks", onclick="opentab(event, 'Contact')">Contact</button>

      </div>
      </div>


			          <div class="header">


<div id="Research Articles" class="tabcontent", style="display:none">

  <h2>Research Articles</h2>

  

<h4>First-author work:</h4>

<p><em> Deciphering causal genomic templates of complex molecular phenotypes </em> <br> SSB*, Anna Seigal, and Juan Caicedo<br>
    <a href = "https://www.biorxiv.org/content/10.1101/2022.08.15.503769v1">preprint, August 2022</a>  
	<br>(corresponding author)
</p>

  <p><em> Tissue Schematics map the specialization of immune tissue motifs and their appropriation by tumors</em> <br> SSB*, G. Barlow* et al.,<br>
    <a href = "bhate et al. 2021 tissue schematics.pdf"> Cell Systems, October 2021</a>  
</p>

<p> <em>Highly multiplexed tissue imaging using repeated oligonucleotide exchange reaction</em> <br> J. Kennedy-Darling*, SSB*, J. Hickey* et al.<br> 
<a href="https://onlinelibrary.wiley.com/doi/full/10.1002/eji.202048891"> European Journal of Immunology, January 2021 </a> </p>

<p><em> Coordinated cellular neighborhoods orchestrate antitumoral immunity at the colorectal cancer invasive front</em> <br> C. Sch&#252;rch*, SSB*, G. Barlow*, D. Phillips* et al.<br> 
<a href="https://www.sciencedirect.com/science/article/pii/S0092867420308709"> Cell, September 2020 </a> </p>

<p><em> Unsupervised representation learning to interrogate cellular behaviours in 47-parameter single-cell resolution tissue imaging data </em> <br> SSB et al.<br> 
<a href="nn2.pdf"> NeurIPS Computational Biology Workshop, December 2017 </a> </p>

<p> * = equal contribution </p>

<h4>Other publications:</h4>
	


<p><em> A tissue atlas of ulcerative colitis to guide TNFinhibitor therapy</em> <br> A. Mayer et al.<br> 
<a href="https://www.science.org/doi/10.1126/sciadv.add1166">Science Advances, January 2023</a>. </p>

<p><em> Splenic red pulp macrophages provide a niche for CML stem cells and induce therapy resistance</em> <br> E. Buhrer et al. <br> 
<a href="https://www.nature.com/articles/s41375-022-01682-2"> Leukemia, September 2022</a>. </p>


<p><em> Immunotherapy of glioblastoma explants induces interferon-&gamma; responses and spatial immune cell rearrangements in tumor center, but not periphery</em> <br> T. Shekarian  et al.<br> 
<a href="https://www.biorxiv.org/content/10.1101/2022.01.19.474897v1.abstract">Science Advances, July 2022</a>. </p>
	
	
  <p><em> CellSeg: a robust, pre-trained nucleus segmentation and pixel quantification software for highly multiplexed fluorescence images </em> <br>
    M. Li, J. Bedia et al. <br>
 <a href="https://link.springer.com/article/10.1186/s12859-022-04570-9">
BMC Bioinformatics, January 2022</a>.</p>

<p><em>Immune cell topography predicts response to PD-1 blockade in cutaneous T cell lymphoma</em> <br> D. Phillips et al. <br> 
  <a href="https://www.nature.com/articles/s41467-021-26974-6">
    Nature Communications, November 2021</a>.</p>

<p><em> Subcellular localization of drug distribution by super-resolution ion beam imaging</em> <br> X. Rovira-Clave, S. Jiang et al.<br> 
  <a href="https://www.nature.com/articles/s41467-021-24822-1">  Nature Communications, July 2021</a>. </p>

<p><em> Landscape of coordinated immune responses to H1N1 challenge in humans</em> <br> Z. Rahil et al.<br> 
<a href="https://www.jci.org/articles/view/1372655"> Journal of Clinical Investigation, January 2021 </a> </p>


<p><em> CRISPR activation screens systematically identify factors that drive neuronal fate and reprogramming</em>  <br> Y. Liu, C. Yu, T. Daley et al.<br> 
<a href="https://www.sciencedirect.com/science/article/pii/S1934590918304405"> Cell Stem Cell, November 2018 </a> </p>

<p><em> Deep profiling of mouse splenic architecture with CODEX multiplexed imaging </em> <br> Y. Goltsev, N. Samusik et al.<br> 
  <a href="https://www.sciencedirect.com/science/article/pii/S0092867418309048"> Cell, September 2018 </a> </p>

<p><em>A multi-modal neural network for learning cis and trans regulation of stress response in yeast </em><br> B. Liu et al.<br> 
<a href="https://www.medrxiv.org/content/10.1101/2020.12.06.20244913vhttps://arxiv.org/abs/1908.09426">
NeurIPS Computational Biology Workshop, December 2017</a>.</p>


<h4>Other preprints:</h4>




<h4>PhD thesis:</h4>

<p> <a href="https://purl.stanford.edu/yc661nd1828"> Towards semantic representations of tissue organization from high-parameter imaging data <a> 2021. </p>

</div>

<div id="Other Work" class="tabcontent", style="display:none">

<h2>Other Work</h2>


<h4> Synthetic biology: </h4>
<p> <a href="https://www.atum.bio/resources/archive/leap-in-transposase"> Leap-in transposases </a> <br> I spent Spring 2015 at Atum (then DNA 2.0) engineering these in yeast using libraries, with Jeremy Minshull et al. </p>

<p> <a href="https://dspace.mit.edu/handle/1721.1/96069"> BBF RFC 106: A Standard Type IIS Syntax for Plants. </a> <br> I worked on making plant cloning easier with the Cambridge-JIC iGEM team 2014.</p>

<p> <a href="http://2014.igem.org/Team:Cambridge-JIC/Marchantia"> Cambridge-JIC iGEM team 2014 </a> <br> We tried to engineer <em> M. polymorpha </em> to express chromoproteins. It was challenging. </p>

<h4> Mathematics: </h4>

<p> <a href="masters.pdf"> My master's thesis (Part III Essay, June 2014): Statistical Properties of Geometric Flows and Equidistribution</a> <br> with Vlad Markovic FRS.</p>


  
</div>


<div id="Activities" class="tabcontent", style="display:none">

<h2>Activities</h2>
<h4>Current:</h4>
<ul class="pull-left">
	<li> Co-Chair of the Broad's Models, Inference and Algorithms initiative Steering Committee,  January 2023- </li>
	<li> Steering Committee member of the Broad's Models, Inference and Algorithms initiative,  September 2022- </li>
	</ul>
<h4>Previous:</h4>



<ul class="pull-left">
<li> Invited speaker, Medical Population Genetics Primer, Broad Institute, February 2023.</li>
<li> Invited speaker, AI/ML Community of Practice Seminar, Takeda, January 2023.</li>
<li> Invited speaker, Thought-Leaders in AI Symposium, Takeda, November 2022.</li>
<li> Invited speaker, Cellular and Tissue Genomics, Genentech, November 2022.</li>
<li> Invited speaker, Data Science department, IISER Pune, October  2022.</li>	
<li> Invited speaker, Models, Inference and Algorithms initiative, Broad Institute, March 2022. <a href="https://www.broadinstitute.org/talks/towards-semantic-representations-tissue-organization-high-parameter-imaging-data">Recording.</a>
</li>
  
        <li> NeurIPS, Long Island CA, 2017: two extended abstracts accepted as posters for the computational biology workshop. 
</li>
        <li> Keystone Single Cell 'Omics, Stockholm 2017: poster. SBSA travel award of $1000.  </li>
	<li> AACR-CIMT-EATI-CRI International Cancer Immunotherapy Conference, NYC 2016: Awarded short talk. BioX Oral Presentation award, $500 + SBSA travel award,  $1000.  </li>
	<li> iGEM, Boston 2014 with the Cambridge-JIC team. Gold medal.  </li>
	</ul>
</div>

<div id="Teaching" class="tabcontent", style="display:none">

<h2>Teaching</h2>

<p> Spring 2018: TA for <em>Ethics of Biotechnology</em> with Prof. David Magnus and Prof. Russ Altman, Stanford University. <br> Hoefer award for mentorship of student writers. </p>

<p> Spring 2017: TA for <em>Synthetic Biology undergraduate lab </em> with Prof. Stanley Qi, Stanford University.</p>

</div>


<div id="Contact" class="tabcontent", style="display:none">
<h2>Contact</h2>
	<ul>
	  <li> Pronouns: he/him </li>
	  <li> Email: my first initial "my last name" at broadinstitute dot org </li>
	<li> Location: Broad Institute  </li> </ul>
</div>


				  </div>
				  </div>


<!-- </div> -->


<script>
function opentab(evt, name) {
  var i, tabcontent, tablinks;
  tabcontent = document.getElementsByClassName("tabcontent");
  for (i = 0; i < tabcontent.length; i++) {
    tabcontent[i].style.display = "none";
  }
  tablinks = document.getElementsByClassName("tablinks");
  for (i = 0; i < tablinks.length; i++) {
    tablinks[i].className = tablinks[i].className.replace(" active", "");
  }
  document.getElementById(name).style.display = "block";
  evt.currentTarget.className += " active";
}
</script>

  </body>
</html>
